<launch>
    <!-- Configure the name of the ROS node -->
    <arg name="node_name" default="depthai_node" />

    <!-- For debugging it is possible to configure a launch prefix -->
    <!-- <arg name="launchPrefix" value="gdb -ex 'catch throw' -ex run \-\-args" /> -->
    <!-- <arg name="launchPrefix" value="valgrind" /> -->
    <arg name="launchPrefix" value="" />

    <!-- Launch the stereo node -->
    <node pkg="depthai_node" type="depthai_node" name="$(arg node_name)" output="screen" launch-prefix="$(arg launchPrefix)">

        <param name="cmd_file" type="string" value="$(find depthai_node)/data/depthai.cmd"/>
        <param name="calibration_file" type="string" value="$(find depthai_node)/data/default.calib" />
        <param name="blob_file" type="string" value="$(find depthai_node)/data/mobilenet_ssd.blob" />
        <param name="blob_config_file" type="string" value="$(find depthai_node)/data/object_detection.json" />
        <param name="metaout" type="bool" value="true" />
        <param name="previewout" type="bool" value="true" />
        <param name="left" type="bool" value="false" />
        <param name="right" type="bool" value="false" />
        <param name="disparity" type="bool" value="false" />
        <param name="depth_mm_h" type="bool" value="false" />
        <param name="depth_color_h" type="bool" value="false" />
    </node>
</launch>
